contracts/test/integration/vip/vip13.sol

```diff
@@ -0,0 +1,484 @@
+    PriceBoundPSM public voltV2UsdcPriceBoundPSM;
+
+    VoltV2 public voltV2;
+
+    VoltMigrator public voltMigrator;
+    MigratorRouter public migratorRouter;
+
+    ITimelockSimulation.action[] private proposal;
+
+    uint256 voltDaiFloorPrice = 9_000;
+    uint256 voltDaiCeilingPrice = 10_000;
+
+    uint256 voltUsdcFloorPrice = 9_000e12;
+    uint256 voltUsdcCeilingPrice = 10_000e12;
+
+    /// @notice target token balance for the DAI PSM to hold
+    uint248 private constant targetBalanceDai = 100_000e18;
+
+    /// @notice target token balance for the USDC PSM to hold
+    uint248 private constant targetBalanceUsdc = 100_000e6;
+
+    /// @notice scale up USDC value by 12 decimals in order to account for decimal delta
+    /// and properly update the buffer in ERC20Allocator
+    int8 private constant usdcDecimalNormalizer = 12;
+
+    /// @notice no scaling to do on DAI as decimals are 18
+    int8 private constant daiDecimalNormalizer = 0;
+
+    ERC20Allocator private allocator =
+        ERC20Allocator(MainnetAddresses.ERC20ALLOCATOR);
+
+    ERC20CompoundPCVDeposit private daiDeposit =
+        ERC20CompoundPCVDeposit(MainnetAddresses.COMPOUND_DAI_PCV_DEPOSIT);
+    ERC20CompoundPCVDeposit private feiDeposit =
+        ERC20CompoundPCVDeposit(MainnetAddresses.COMPOUND_FEI_PCV_DEPOSIT);
+    ERC20CompoundPCVDeposit private usdcDeposit =
+        ERC20CompoundPCVDeposit(MainnetAddresses.COMPOUND_USDC_PCV_DEPOSIT);
+
+    constructor() {}
+
+    function getMainnetProposal()
+        public
+        override
+        returns (ITimelockSimulation.action[] memory prop)
+    {
+        voltV2 = new VoltV2(MainnetAddresses.CORE);
+        voltMigrator = new VoltMigrator(MainnetAddresses.CORE, address(voltV2));
+        deployPsms(address(voltV2));
+
+        migratorRouter = new MigratorRouter(
+            address(voltV2),
+            address(voltMigrator),
+            voltV2DaiPriceBoundPSM,
+            voltV2UsdcPriceBoundPSM
+        );
+
+        address[] memory toWhitelist = new address[](3);
+        toWhitelist[0] = address(voltV2UsdcPriceBoundPSM);
+        toWhitelist[1] = address(voltV2DaiPriceBoundPSM);
+        toWhitelist[2] = address(voltMigrator);
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.CORE,
+                arguments: abi.encodeWithSignature(
+                    "grantMinter(address)",
+                    MainnetAddresses.TIMELOCK_CONTROLLER
+                ),
+                description: "Grant minter role to timelock"
+            })
+        );
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: address(voltV2),
+                arguments: abi.encodeWithSignature(
+                    "mint(address,uint256)",
+                    address(voltMigrator),
+                    voltInDaiPSM + voltInUsdcPSM
```

Comment:
This number is incorrect. I believe we should mint the current total volt supply into the migrator contract so that all current users can migrate over.